src/core/adapters/github.ts

```diff
@@ -38,16 +38,21 @@ async function attempt(
   if ($has(select.issuesPage, doc)) {
     const issues = $all(select.issuesPage, doc);
 
-    const tickets = issues.map((issue) => {
+    const tickets = issues.reduce((acc, issue) => {
       const id = $value("input.js-issues-list-check", issue);
       const title = $text("a.js-navigation-open", issue);
+
+      if (!id || !title) return acc;
```

Comment:
The Typescript compiler complained, that we cannot have `Null` titles in `TicketData`. That's actually correct and I don't know why it wasn't a problem before...